Changelog Updates (#1733)
forcedotcom/salesforcedx-vscode: packages/salesforcedx-vscode/CHANGELOG.md

# 47.3.0 - October 24, 2019

> ## Added

Reviewer: FYI this PR #1730 might still make it into tomorrow's release. I'm just waiting on some cli tests.

Reviewer 2: This may also make it into tomorrow's release. #1741

#### salesforcedx-vscode-core

> - Enable diff command for layouts and permsets ([PR #1713](https://github.com/forcedotcom/salesforcedx-vscode/pull/1713))

Reviewer: ```suggestion
- Enable diff for layouts and permission sets in `SFDX: Diff File Against Org` command ([PR #1713](https://github.com/forcedotcom/salesforcedx-vscode/pull/1713))
```

## Fixed

#### docs

> - Fix Japanese user guide section (#1720) ([PR #1719](https://github.com/forcedotcom/salesforcedx-vscode/pull/1719))

Reviewer: We can remove this one since we did the fix during last week's release as part of releasing the Japanese translations.

Author: @lcampos Are you certain? If I'm reading the logs correctly it was merged after 47.2.0.

Reviewer: Yes, we had to do it straight to the `master` branch because we found it after the release. We then ported it back to `develop`.

Author: Okay, thanks, I'll remove it.

Reviewer 2: Just an fyi, that was a real one-off scenario

# 47.3.0 - October 24, 2019

## Added

#### salesforcedx-vscode-core

- Enable diff for layouts and permission sets in `SFDX: Diff File Against Org` command ([PR #1713](https://github.com/forcedotcom/salesforcedx-vscode/pull/1713))

- Added the `analytics` template option to the `SFDX: Project Create` command ([PR #1730](https://github.com/forcedotcom/salesforcedx-vscode/pull/1730))

#### salesforcedx-vscode-apex

> - Open beta for SOQL Language Server ([PR #1741](https://github.com/forcedotcom/salesforcedx-vscode/pull/1741)) that provides code completion suggestions for your SOQL queries.

Reviewer: ```suggestion
- Open beta for SOQL Language Server that provides code completion suggestions for SOQL queries ([PR #1741](https://github.com/forcedotcom/salesforcedx-vscode/pull/1741))
```

#### docs

> - Added documentation for SOQL language server ([PR #1741](https://github.com/forcedotcom/salesforcedx-vscode/pull/1741))

Reviewer: ```suggestion
- [Write SOQL Queries](https://forcedotcom.github.io/salesforcedx-vscode/articles/soql/writing) documentation for SOQL language server ([PR #1741](https://github.com/forcedotcom/salesforcedx-vscode/pull/1741))
```
